examples/nextjs-14-sensitive-info/app/api/arcjet/route.ts

```diff
@@ -0,0 +1,36 @@
+import arcjet, { sensitiveInfo } from "@arcjet/next";
+import { NextResponse } from "next/server";
+
+const aj = arcjet({
+  // Get your site key from https://app.arcjet.com
+  // and set it as an environment variable rather than hard coding.
+  // See: https://nextjs.org/docs/app/building-your-application/configuring/environment-variables
+  key: process.env.ARCJET_KEY!,
+  rules: [
```

Comment:
Please add shield. We try to add shield in every SDK example because we originally ran it implicitly but now it must be configured.